apio-consumer/src/main/kotlin/com/liferay/apio/consumer/parser/ThingParser.kt

```diff
@@ -55,11 +50,15 @@ class ThingParser {
 				?: listOf()
 		}
 
+		@JvmStatic
+		@Throws(JsonSyntaxException::class)
+		fun String.toJsonMap(): Map<String, Any> {
```

Comment:
I like extension methods but I think in this case you are adding a too specific behavior to a String object. I won't use an extension method in this case.

Reply:
Yeah, I agree, I fixed in the last commit.